dev/style.scss

```diff
@@ -54,6 +54,9 @@ img {
   margin-bottom: 20px;
 }
 
+[data-eta-trigger] {
```

Comment:
Why do we need this? And is this related to "add prevent-default option"?

Reply:
I just want to make the permalink be `block` (to have a margin-bottom CSS property)